crates/forge/src/run_tests/workspace.rs

```diff
@@ -32,6 +33,10 @@ pub async fn run_for_workspace(args: TestArgs) -> Result<ExitStatus> {
         can_coverage_be_generated(&scarb_metadata)?;
     }
 
+    if is_enabled() {
+        can_be_generated(&scarb_metadata)?;
```

Comment:
```suggestion
    if backtrace::is_enabled() {
        backtrace::can_be_generated(&scarb_metadata)?;
```
This is for better readability. Alternatively, you can rename these functions to `is_backtrace_enabled` and `can_backtrace_be_generated`.